contracts/utils/addresses.js

```diff
@@ -61,6 +61,9 @@ addresses.mainnet.CVXRewardsPool = "0x7D536a737C13561e0D2Decf1152a653B4e615158";
 addresses.mainnet.CVXLocker = "0x72a19342e8F1838460eBFCCEf09F6585e32db86E";
 // Frax
 addresses.mainnet.FXS = "0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0";
+addresses.mainnet.Frax = "0x853d955acef822db058eb8505911ed77f175b99e";
+addresses.mainnet.FraxCommunityBags =
```

Comment:
Is this used somewhere?

Reply:
Yes in the _fixture to fund the FXS tokens. Those can unfortunately not be gained by either calling `mint` on the FXS contract or by altering the storage slots. Since right after the slot manipulation / minting the transfer fails because vote count check triggers.